client/reader/site-stream/empty.jsx

```diff
@@ -45,7 +45,7 @@ const SiteEmptyContent = ( { translate } ) => {
 			onClick={ recordSecondaryAction }
 			href="/read/search"
 		>
-			{ translate( 'Find Sites to Follow' ) }
+			{ translate( 'Find sites to follow' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.